docs/docs/01-ibc/04-middleware/01-overview.md

```diff
@@ -1,23 +1,32 @@
-<!--
-order: 1
--->
+---
+title: IBC middleware
+sidebar_label: IBC middleware
+sidebar_position: 1
+slug: /ibc/middleware/overview
+---
 
 # IBC middleware
 
-Learn how to write your own custom middleware to wrap an IBC application, and understand how to hook different middleware to IBC base applications to form different IBC application stacks {synopsis}.
+:::note Synopsis
+Learn how to write your own custom middleware to wrap an IBC application, and understand how to hook different middleware to IBC base applications to form different IBC application stacks
+:::
 
 This documentation serves as a guide for middleware developers who want to write their own middleware and for chain developers who want to use IBC middleware on their chains.
 
 After going through the overview they can consult respectively:
 
-- [documentation on developing custom middleware](develop.md)
-- [documentation on integrating middleware into a stack on a chain](integration.md)
+- [documentation on developing custom middleware](02-develop.md)
+- [documentation on integrating middleware into a stack on a chain](03-integration.md)
+
+:::note
 
 ## Pre-requisite readings
```

Comment:
Noticed that some headings are "Pre-requisite readings" and some are "Pre-requisites Readings". Not 100% what casing is desired, IIRC we had been following some google style guide for markdown docs which suggested to use the former above.